### PR TITLE
Performance and test improvement for LearnerMetricsViewSet

### PR DESCRIPTION
**Pull request by johnbaldwin**
https://appsembler.atlassian.net/browse/RED-1261

* Added prefetch_related to the User model in LearnerMetricsViewSet to query the related "CourseEnrollment" records
* Updated test coverage to include users, courses, and enrollments for two sites in order to ensure we are retrieving only the data we want and avoid "site bleed". For this, created an initial generalized fixture in "tests/conftest.py". This initial fixture contains User, CourseOverview, CourseEnrollment, Organization, and Site records. If the version of "organizations" is our fork (supports multi-site) then OrganizationCourse and UserOrganizationMapping records are also filled
* Updated devsite to support multi-tenant mode (Appsembler's organizations fork). For this, added `django-environ` to be able to set multi-tenant mode in the `devsite/devsite/.env` file. This is the default location `django-environ` seeks

This commit lays the groundwork for additional performance imporovement to reduce the number of queries needed in the serializers used for LearnerMetricsViewSet

### CODE REVIEW

**Comment by OmarIthawi** [comment]
@johnbaldwin is this ready for review? I see that you're still pushing new big changes.

**Comment by johnbaldwin** [comment]
Looks like codecov got stuck again

**Comment by johnbaldwin** [comment]
@OmarIthawi This should be ready for review now. Not sure why codecov is getting stuck again

**Comment by johnbaldwin** [comment]
@OmarIthawi This is ready for review, please. I merged in the Ginkgo fix.

File: figures/serializers.py [hunk 1]
```diff
@@ -882,7 +866,7 @@ class LearnerMetricsSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = get_user_model()
-        list_serializer_class = LearnerMetricsListSerializer
+        # list_serializer_class = LearnerMetricsListSerializer
```
Comment (OmarIthawi) [inline figures/serializers.py]:
```suggestion
```

File: figures/serializers.py [hunk 2]
```diff
@@ -876,14 +868,14 @@ def get_enrollments(self, user):
         Rely on the caller (the view) to filter users and prefetch related
         """
 
-        user_enrollments = user.courseenrollment_set.all()
+        # user_enrollments = user.courseenrollment_set.all()
```
Comment (OmarIthawi) [inline figures/serializers.py:871]:
```suggestion
```

**Review by OmarIthawi — APPROVED** [review]

File: figures/serializers.py (hunk 2 above)
Comment (johnbaldwin) [reply figures/serializers.py:871]: I'll remove the dead code. Thanks

**Pull request merged**